### PR TITLE
[Indigo] Indigo simulation updates

### PR DESCRIPTION
**Pull request by fmessmer**
prepare all cob3-X components for indigo simulation

### CODE REVIEW

**Comment by fmessmer** [comment]
merge together with: 
- [schunk_modular_robotics#129](https://github.com/ipa320/schunk_modular_robotics/pull/129)
- [cob_robots#276](https://github.com/ipa320/cob_robots/pull/276)
- [cob_command_tools#83](https://github.com/ipa320/cob_command_tools/pull/83)
- [cob_common#144](https://github.com/ipa320/cob_common/pull/144)
- [cob_control#25](https://github.com/ipa320/cob_control/pull/25)
- [cob_simulation#78](https://github.com/ipa320/cob_simulation/pull/78)

File: schunk_description/urdf/lwa/lwa.urdf.xacro [hunk 1]
```diff
@@ -39,23 +33,18 @@
...
       <origin xyz="0 0 -0.250" rpy="0 0 3.1415926"/>
-      <parent link="${name}_0_link"/>
+      <parent link="${name}_base_link"/>
       <child link="${name}_1_link"/>
       <axis xyz="0 0 1"/>
       <calibration rising="${arm_1_ref}"/>
       <dynamics damping="10" />
-      <limit effort="370" velocity="2.0" lower="-6.25" upper="6.25"/>
-      <safety_controller k_position="20" k_velocity="50" soft_lower_limit="${-6.25 + 0.01}" soft_upper_limit="${6.25 - 0.01}" />
+      <limit effort="370" velocity="2.0" lower="-3.1415" upper="3.1415"/>
```
Comment (floweisshardt) [inline schunk_description/urdf/lwa/lwa.urdf.xacro:41]: why did the limits change from 2_pi to 1_pi

File: schunk_description/urdf/lwa/lwa.urdf.xacro [hunk 2]
```diff
@@ -116,18 +97,13 @@
       <axis xyz="0 0 1"/>
       <calibration rising="${arm_3_ref}"/>
       <dynamics damping="5" />
-      <limit effort="176" velocity="2.0" lower="-6.25" upper="6.25"/>
-      <safety_controller k_position="20" k_velocity="25" soft_lower_limit="${-6.25 + 0.01}" soft_upper_limit="${6.25 - 0.01}" />
+      <limit effort="176" velocity="2.0" lower="-3.1415" upper="3.1415"/>
```
Comment (floweisshardt) [inline schunk_description/urdf/lwa/lwa.urdf.xacro:100]: why did the limits change from 2pi to 1pi

File: schunk_description/urdf/lwa/lwa.urdf.xacro [hunk 3]
```diff
@@ -188,17 +156,13 @@
       <axis xyz="0 0 1" />
       <calibration rising="${arm_5_ref}"/>
       <dynamics damping="5" />
-      <limit effort="41.6" velocity="2.0" lower="-6.25" upper="6.25" />
-      <safety_controller k_position="20" k_velocity="25" soft_lower_limit="${-6.25 + 0.01}" soft_upper_limit="${6.25 - 0.01}" />
+      <limit effort="41.6" velocity="2.0" lower="-3.1415" upper="3.1415" />
```
Comment (floweisshardt) [inline schunk_description/urdf/lwa/lwa.urdf.xacro:159]: why did the limits change from 2pi to 1pi

File: schunk_description/urdf/lwa/lwa.urdf.xacro [hunk 4]
```diff
@@ -258,17 +215,13 @@
       <axis xyz="0 0 1" />
       <calibration rising="${arm_7_ref}"/>
       <dynamics damping="5" />
-      <limit effort="9.6" velocity="2.0" lower="-6.25" upper="6.25" />
-      <safety_controller k_position="20" k_velocity="25" soft_lower_limit="${-6.25 + 0.01}" soft_upper_limit="${6.25 - 0.01}" />
+      <limit effort="9.6" velocity="2.0" lower="-3.1415" upper="3.1415" />
```
Comment (floweisshardt) [inline schunk_description/urdf/lwa/lwa.urdf.xacro:218]: why did the limits change from 2pi to 1pi

File: schunk_description/urdf/lwa4p_extended/lwa4p_extended.urdf.xacro [hunk 5]
```diff
@@ -80,7 +80,7 @@
       <axis xyz="0 0 1"/>
       <calibration rising="${arm_1_ref}"/>
       <dynamics damping="0" friction="0"/>
-      <limit effort="81.5" velocity="0.42" lower="${-4.712 + safety_offset}" upper="${4.712 - safety_offset}"/>   <!--Theoretical limit +/-270 , Real Limit +/- 270-->
+      <limit effort="81.5" velocity="0.66" lower="${-4.712 + safety_offset}" upper="${4.712 - safety_offset}"/>   <!--Theoretical limit +/-270 , Real Limit +/- 270-->
```
Comment (floweisshardt) [inline schunk_description/urdf/lwa4p_extended/lwa4p_extended.urdf.xacro:83]: why did the velocity limit change?

File: schunk_description/urdf/lwa4p_extended/lwa4p_extended.urdf.xacro [hunk 6]
```diff
@@ -117,7 +117,7 @@
       <axis xyz="0 0 1"/>
       <calibration rising="${arm_2_ref}"/>
       <dynamics damping="0" friction="0"/>
-      <limit effort="48" velocity="1.2566" lower="${-1.92 + safety_offset}" upper="${1.92 - safety_offset}"/>   <!--Theoretical limit +/-170 , Real Limit +/- 110 (self collision)-->
+      <limit effort="48" velocity="1.57" lower="${-1.91 + safety_offset}" upper="${1.91 - safety_offset}"/>   <!--Theoretical limit +/-170 , Real Limit +/- 110 (self collision)-->
```
Comment (floweisshardt) [inline schunk_description/urdf/lwa4p_extended/lwa4p_extended.urdf.xacro:120]: why did the limits change? ...same for all joints...

**Comment by floweisshardt** [comment]
velocity limits updates are measured on hardware: OK
position limits updates are due to MoveIt limitation with large ranges

**Comment by floweisshardt** [comment]
visual review:OK

**Comment by fmessmer** [comment]
Yes, this PR includes #127

**Comment by NadiaHG** [comment]
Visual Review: OK

**Comment by floweisshardt** [comment]
visual review: OK

**Pull request merged**